aii-core/pom.xml

```diff
@@ -207,6 +207,7 @@
             <requires>
               <require>perl-CDB_File</require>
               <require>perl-XML-Simple</require>
+              <require>perl-Parallel-ForkManager</require>
```

Comment:
why is there any `require` at all? without requires, the perl `use` lines get translated in dependencies.